optuna_integration/skorch.py

```diff
@@ -0,0 +1,46 @@
+from typing import Any
+
+import optuna
+
+
+with optuna._imports.try_import() as _imports:
```

Comment:
Please use `optuna-integration` side `_imports`.

Reply:
I have fixed it.